src/PublicSuffixListManager.php

```diff
@@ -87,7 +127,7 @@ public function refreshPublicSuffixList()
      *
      * @return int Number of bytes that were written to the file
      */
-    public function fetchListFromSource(): int
+    protected function fetchListFromSource(): int
```

Comment:
Nice catch. Any chance we can now change all the `protected` to `private`?

Reply:
done